openfe/utils/visualization.py

```diff
@@ -115,20 +118,26 @@ def _draw_molecules(
     highlight_color: Tuple[float, float, float, float]
         RGBA tuple for the default highlight color used in the mapping
         visualization
+    atom_mapping: Dict[Tuple[int,int], Dict[int, int]], optional
+        used to align the molecules to each othter for clearer visualization,
+        default None
     """
     if d2d is None:
         # select default layout based on number of molecules
-        grid_x, grid_y = {
-            1: (1, 1),
-            2: (2, 1),
-        }[len(mols)]
-        d2d = Chem.Draw.rdMolDraw2D.MolDraw2DCairo(grid_x*300, grid_y*300,
-                                                   300, 300)
+        grid_x, grid_y = {1: (1, 1), 2: (2, 1), }[len(mols)]
+        d2d = Chem.Draw.rdMolDraw2D.MolDraw2DCairo(
+            grid_x * 300, grid_y * 300, 300, 300)
 
     # squash to 2D
-    copies = [copy.deepcopy(mol) for mol in mols]
-    for mol in copies:
-        AllChem.Compute2DCoords(mol)
+    copies = [Chem.Mol(mol) for mol in mols]
+    [AllChem.Compute2DCoords(mol) for mol in copies]
```

Comment:
[don't use a list comprehension unless you're assigning it](https://treyhunner.com/2019/03/abusing-and-overusing-list-comprehensions-in-python/#Loops_disguised_as_comprehensions)